src/Stackctl/StackSpecYaml.hs

```diff
@@ -86,37 +99,40 @@ parametersYaml :: [ParameterYaml] -> ParametersYaml
 parametersYaml = ParametersYaml
 
 data ParameterYaml = ParameterYaml
-  { _pyKey :: Key
-  , _pyValue :: Maybe ParameterValue
+  { pyKey :: Key
+  , pyValue :: Maybe ParameterValue
   }
+  deriving stock (Eq, Show)
 
 parameterYaml :: Parameter -> Maybe ParameterYaml
 parameterYaml p = do
   k <- p ^. parameter_parameterKey
   pure
     $ ParameterYaml (Key.fromText k)
     $ ParameterValue
+    . Last
     <$> p
     ^. parameter_parameterValue
 
 unParameterYaml :: ParameterYaml -> Parameter
 unParameterYaml (ParameterYaml k v) =
-  makeParameter (Key.toText k) $ unParameterValue <$> v
+  makeParameter (Key.toText k) $ getLast . unParameterValue <$> v
 
 instance FromJSON ParameterYaml where
   parseJSON = withObject "Parameter" $ \o ->
     (ParameterYaml <$> o .: "Name" <*> o .:? "Value")
       <|> (ParameterYaml <$> o .: "ParameterKey" <*> o .:? "ParameterValue")
 
 newtype ParameterValue = ParameterValue
-  { unParameterValue :: Text
+  { unParameterValue :: Last Text
   }
-  deriving newtype ToJSON
+  deriving stock (Eq, Show)
+  deriving newtype (Semigroup, ToJSON)
 
 instance FromJSON ParameterValue where
   parseJSON = \case
-    String x -> pure $ ParameterValue x
-    Number x -> pure $ ParameterValue $ dropSuffix ".0" $ pack $ show x
+    String x -> pure $ ParameterValue $ Last x
```

Comment:
this surprises me; not wrong or even bad per se, but sticking the last in here just to have the correct semigroup behavior when deriving a newtype is really surprises me. If I looked at this without having saw this PR, i'd be really confused as to why this was constructed in this way. 

Up to you if you want to change anything, just my 2c